gitlab/__init__.py

```diff
@@ -521,3 +521,21 @@ def Team(self, id=None, **kwargs):
         to write it on the server.
         """
         return Team._get_list_or_object(self, id, **kwargs)
+
+    def Status(self, id=None, **kwargs):
```

Comment:
I plan to deprecate the similar functions in the Gitlab class (it's already done in the other classes), so I'm not sure this is useful to add this one. Could you also add a manager attribute in the constructor?